### DISCUSSION TITLE
Method of embeding cables in walls like AE2's Facades

### DISCUSSION THREAD

**Discussion by Astralchroma** [TechReborn/TechReborn#3194] [Ideas]
see title

**Comment by xanthian** [comment]
already exists. 

see wooden plate & painting tool.

**Reply by Astralchroma** [reply]
Didn't notice that, perhaps it needs to be documented better? Perhaps on the [Cables](https://wiki.techreborn.ovh/doku.php?id=energy:cables) page?
